webtool/test_correct.py

```
@@ -2189,3 +2189,57 @@ def ask_llm_verboten(*a, **kw):
+    """
+    import builtins
+    from webtool import llm as _llm
+
+    _glossar_projekt(monkeypatch, tmp_path)
+    monkeypatch.setattr(_llm, "use_api", lambda: True)
+    monkeypatch.setattr(_llm, "complete",
+                        lambda *a, **kw: '{"proper_nouns": [], "likely_corrections": []}')
+
+    echt_open = builtins.open
+
+    def open_mit_marke(datei, *a, **kw):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `open_mit_marke`

(ANN202)